fast_setup.py

```diff
@@ -56,11 +63,16 @@ def agent_prompt_suffix(suffix, cat):
 {{declarative_memory}}
 
 {{tools_output}}
+"""
 
-ALWAYS answer in {settings["language"]}!
+    if settings["language"]:
```

Comment:
Probably this is always defined.
We should check if it is empty (empty string)

Reply:
Agree, don't know if it makes sense, but instead of having a text input field, we could place a dropdown with default values. Among these there could be "None", and if  `language != "None"` we add the string in the suffix

Reply:
Ok i will do it tommorow!